Fibonacci.java

```diff
@@ -0,0 +1,18 @@
+public class Fibonacci {
+
+	public static void main(String[] args) { 
+		int nthFib = Integer.parseInt(args[0]);
+		System.out.println(new Fibonacci().findNthFib(nthFib));
```

Comment:
change new instance name to something other than the class name:
new Fibonacci()